examples/coiled/aws/coiled-add-asarray.py

```diff
@@ -21,7 +21,7 @@
     c = xp.add(a, b)
     res = c.compute(
         executor=executor,
-        memory="1 GiB",  # must be greater than allowed_mem
+        memory=["1 GiB", "8 GiB"] # memory range
```

Comment:
Looks like there's a missing comma here?

Also might be worth preserving the previous comment too:

```
# memory range, lower value must be at least allowed_mem"
```

Reply:
I didn't realise we could submit a range now, that's nice

Reply:
To be clear - the `memory` argument is passed through to the Coiled function, it's not interpreted by Cubed.